sync/listen.py

```diff
@@ -188,13 +190,14 @@ def accept(self, body):
 
 class GitHubFilter(Filter):
     name = "github"
-    event_filters = {item: lambda x: True for item in handlers.GitHubHandler.dispatch_event.keys()}
+    event_filters = {item: lambda x: True
+                     for item in handlers.GitHubHandler.dispatch_event.keys()}
     event_filters["status"] = lambda x: x["payload"]["context"] != "upstream/gecko"
     event_filters["push"] = lambda x: x["payload"]["ref"] == "refs/heads/master"
 
     def __init__(self, config, logger):
         super(GitHubFilter, self).__init__(config, logger)
-        repo_path = urlparse.urlparse(config["web-platform-tests"]["repo"]["url"]).path
+        repo_path = urllib.parse.urlparse(config["web-platform-tests"]["repo"]["url"]).path
```

Comment:
This looks a better way to do it than before. In other parts of the code, the import to `six` spray `six.` keywords into the code instead of the ready to go python 3 value.

It could be a reminder to remove them when full python 3 conversion is reached or like here just the import line to change at the top.